src/components/student-dashboard/events-panel.cjsx

```diff
@@ -46,7 +46,7 @@ module.exports = React.createClass
         <BS.Col xs={5} xsOffset={2} smOffset={0} sm={2} className='progress-label'>
           Progress
         </BS.Col>
-        <BS.Col xs={5} sm={2} className='due-at-label'>Due</BS.Col>
+        <BS.Col xs={5} sm={2} className='due-at-label'>Due (7:00am)</BS.Col>
```

Comment:
I think it's fine to wait until the teacher can set the time; just was confirming BE does not yet send that time ; )